src/actions/index.js

```diff
@@ -0,0 +1,72 @@
+export function loadMyMovieList(){
+  return function (dispatch){
+    dispatch({
+      type:"LOAD_MY_MOVIE_LIST"
+    });
+    fetch("/movies")
+    .then( (response) => {
+      return response.json();
+    }).then((movies) => {
+      dispatch(myMovieListLoaded(movies));
+    });
+  };
+}
+export function myMovieListLoaded(movies){
+      type:"MY_MOVIE_LIST_LOADED"
+      value:movies
+  }
+export function loadSearch(searchTerm){
+  return function (dispatch){
+  dispatch({
+    type:"LOAD_SEARCH"
+  });
+  fetch("https://api.themoviedb.org/3/search/multi?query=searchTerm&api_key=18da039951b8d9f69a46f0d0b6604c46")
```

Comment:
Double check this line of code.  searchTerm is supposed to be a variable. RIght now its a hard coded string. The only movie you will ever search for is "searchTerm"

Reply:
got it, I had my suspicions about that..still not getting photos to display in saved movies. out of time today will revisit tomorrow morning